src/app/container/versions/versions.component.ts

```diff
@@ -54,6 +54,10 @@ export class VersionsContainerComponent extends Versions implements OnInit {
     super(dockstoreService, dateService, sessionQuery);
   }
 
+  setDefaultSortColumn(): string {
```

Comment:
`set` has the implication that this method modifies state, which it doesn't. Suggest either plain `defaultSortColumn` or `getDefaultSortColumn`.